x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -710,6 +710,63 @@ class PackagePolicyService {
     }
   }
 
+  public async enrichPolicyWithDefaultsFromPackage(
+    soClient: SavedObjectsClientContract,
+    newPolicy: NewPackagePolicy
+  ): Promise<NewPackagePolicy> {
+    let newPackagePolicy: NewPackagePolicy = newPolicy;
+    if (newPolicy.package) {
+      const newPP = await this.buildPackagePolicyFromPackageWithVersion(
+        soClient,
+        newPolicy.package.name,
+        newPolicy.package.version
+      );
+      if (newPP) {
+        const inputs = newPolicy.inputs.map((input) => {
+          const defaultInput = newPP.inputs.find((i) => i.type === input.type);
```

Comment:
Should we map on `policy_template` too here for granular integration like AWS?

Reply:
In request only `input.type` and `enabled` is mandatory, so this code is mapping the rest of properties from the base package (`streams`, `vars`, etc.).
Could you give an example what you mean by the AWS use case?

Reply:
Sure for example when you try to add the AWS billing integration this we are sending the following payload 
<img width="574" alt="Screen Shot 2021-12-06 at 10 44 54 AM" src="https://user-images.githubusercontent.com/1336873/144876900-77ff1ed3-b1dd-4e9d-bdb2-ab317b58e539.png">

as there is multiple input with the same type, so I think in the case where there is a policy_template on the payload input we should probably use it to find the default input. 

It is clearer? I can rephrase otherwise :)

Reply:
yes, it is clear now, so the enrich should consider `policy_template` too if set
I'll update it.

Reply:
@nchaulet done, please check again when you get a chance